bigml-parallel/src/main.rs

```diff
@@ -170,11 +173,22 @@ async fn resource_id_to_execution(
     // Add tags.
     args.tags = opt.tags.clone();
 
-    // Execute our script, retrying if needed.
-    //
-    // TODO: Add retry logic? Do we need it?
+    // Execute our script, retrying the creation of the execution if needed.
     let client = new_client()?;
-    let mut execution = client.create(&args).await?;
+    let opt = WaitOptions::default()
+        .retry_interval(Duration::from_secs(60))
+        .backoff_type(BackoffType::Exponential)
+        .allowed_errors(6)
+        .timeout(Duration::from_secs(2 * 60 * 60));
```

Comment:
these are reasonable for now, though eventually they should probably be configurable from whatever top-level function is being called

Reply:
This is called directly from `bigml-parallel`'s own code.

It's not currently configurable because if you're getting non-stop "402 Payment Required" for an hour, the natural conclusion is that payment is probably required. :-)